hardhat.config.ts

```diff
@@ -35,6 +36,9 @@ function getChainConfig(chain: keyof typeof chainIds): NetworkUserConfig {
     case "base-sepolia":
       jsonRpcUrl = "https://" + chain + ".g.alchemy.com/v2/" + alchemyApiKey;
       break;
+    case "night-testnet":
+      jsonRpcUrl = "https://rpc-night-testnet-brk51d2fuk.t.conduit.xyz";
```

Comment:
nit: env?

Reply:
i think the env should be ok its the public rpc endpoint

Reply:
actually eh yea ill change it